webpack.mix.js

```diff
@@ -10,20 +14,90 @@ require('laravel-mix-polyfill');
  | Mix provides a clean, fluent API for defining some Webpack build steps
  | for your Mantle application
  |
+ | https://laravel-mix.com/docs/6.0/
+ |
  */
 
-// Let mix know Mantle is in a public path.
+/*
+| --------------------------------------------------------------------------
+| Public Path
+| --------------------------------------------------------------------------
+|
+| Define the public path used by the build process. This must be called before
+| compiling any entries.
+|
+*/
+
 mix.setPublicPath(process.env.APP_BUILD_PATH || 'build');
 
-// Define the entries you need built with mix!
+/*
+| --------------------------------------------------------------------------
+| Discover Entries
+| --------------------------------------------------------------------------
+|
+| Discover entries in the 'entries' directory and automatically compile them.
+| index.js files that live within a folder inside entries will automatically be
+| compiled.
+|
+| Discover index.js files within a specific directory:
+|
+|   mix.discover('entries');
+|
+| Discover index.ts files within a specific directory using Typescript:
+|
+|  mix.discover('entries', { type: 'ts' });
```

Comment:
Could it not detect `ts` files as well as `js` files without specification? Looks like you have the logic here: https://github.com/alleyinteractive/mantle/pull/118/files#diff-0da3eaa1f1380aee7113ee4a1c26d26b1e425f1b03994413a7fdf5b068b654bfR49

I am not that familiar with Mix, so I could be missing something.